src/components/four-column-block.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import Container from 'react-bootstrap/Container'
+import Card from 'react-bootstrap/Card'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+
+import arrow1 from '../images/arrow1.png'
+import arrow2 from '../images/arrow2.png'
+import arrow3 from '../images/arrow3.png'
+import arrow4 from '../images/arrow4.png'
+
+const FourColumnBlock = () => (
+  <Container>
+    <p className="lead text-center font-weight-bold">
+      As a result of this program, we anticipate an increase in:
+    </p>
+
+    <Row>
+      <Col sm={3}>
+        <Card>
+          <Card.Img variant="top" src={arrow1} />
+          <Card.Body>
+            <Card.Text className="font-weight-bold">
+              Some quick example text to build on the card title and make up the
```

Comment:
you can see here there is just placeholder text. this kinds makes this PR incomplete